lib/ui/geometry.dart

```diff
@@ -596,6 +596,16 @@ class Rect {
       ..[3] = math.max(a.dy, b.dy);
   }
 
+  /// Construct the largest finite rectangle.
+  Rect.largest() {
+    const double skScalarMax = 3.402823466e+38;
```

Comment:
document where this number comes from, in case someone has to debug this code one day